python/paddle/fluid/dygraph/dygraph_to_static/logical_transformer.py

```diff
@@ -18,6 +18,30 @@
 from paddle.fluid.dygraph.dygraph_to_static.utils import ast_to_source_code
 
 
+def cmpop_node_to_str(node):
+    if isinstance(node, gast.Eq):
+        return "=="
+    if isinstance(node, gast.NotEq):
+        return "!="
+    if isinstance(node, gast.Lt):
+        return "<"
+    if isinstance(node, gast.LtE):
+        return "<="
+    if isinstance(node, gast.Gt):
+        return ">"
+    if isinstance(node, gast.GtE):
+        return ">="
+    if isinstance(node, gast.Is):
+        return "is"
+    if isinstance(node, gast.IsNot):
+        return "is not"
+    if isinstance(node, gast.In):
+        return "in"
+    if isinstance(node, gast.NotIn):
+        return "not in"
```

Comment:
In next PR，we can consider a dict to config the relationship between `gast.node` and `cmop`, which is O(1).

Or use `if-elif-elif` to avoid execute every conditional branch.